### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by pupara**
Added 1.1.8-preview and 1.1.9-preview releases.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ This page provides the latest Teams PowerShell change log for both General Avail
 
 | Date | Version | Updates |
 |------- | -------------------- | ------------------------------ |
+| November 2020 | [1.1.9-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.9-preview) | <li>Bug fixes and improvements.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| November 2020 | [1.1.9-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.9-preview) | <li>Bug fixes and improvements</li>|
```

File: Teams/teams-powershell-release-notes.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
+| November 2020 | [1.1.8-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.8-preview) | <li>Bug fixes and improvements.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:30]:
```suggestion
| November 2020 | [1.1.8-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.8-preview) | <li>Bug fixes and improvements</li>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Minor edits for consistency.

**Comment by scanum** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**